cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/subnet/selector/AzureSubnetSelectorService.java

```diff
@@ -36,4 +44,18 @@ public SubnetSelectionResult select(Collection<CloudSubnet> subnetMetas, SubnetS
         }
         return Optional.empty();
     }
+
+    public SubnetSelectionResult selectForPrivateEndpoint(Collection<CloudSubnet> subnetMetas, boolean existingNetwork) {
+        List<CloudSubnet> suitableCloudSubnet;
+        if (existingNetwork) {
+            suitableCloudSubnet = subnetMetas.stream()
```

Comment:
A log line here would be nice I think

Reply:
I added a log line to show network registration type (existing / new). Result of selection is now logged in SubnetChooserService.